frb_codegen/src/library/codegen/parser/mir/parser/function/real/mod.rs

```diff
@@ -269,6 +269,25 @@ impl<'a, 'b> FunctionParser<'a, 'b> {
     }
 }
 
+fn compute_dart_async(func: &HirFlatFunction, attributes: &FrbAttributes, default_dart_async: bool) -> bool { // [LARPOUX]
```

Comment:
Your name is already on README "contributors list" ;)

```suggestion
fn compute_dart_async(func: &HirFlatFunction, attributes: &FrbAttributes, default_dart_async: bool) -> bool {
```